index.php

```diff
@@ -8,8 +8,8 @@
 require_once(__DIR__ . '/OutputXMLHelper.php');
 $outputXmlHelper = new OutputXMLHelper();
 // Get URL params
-$start = $outputXmlHelper->getUrlParam($outputXmlHelper::START_NAME, $outputXmlHelper::DEFAULT_START_VALUE, $_GET);
-$count = $outputXmlHelper->getUrlParam($outputXmlHelper::COUNT_NAME, $outputXmlHelper::DEFAULT_COUNT_VALUE, $_GET);
+$start = (int)$outputXmlHelper->getUrlParam($outputXmlHelper::START_NAME, $outputXmlHelper::DEFAULT_START_VALUE, $_GET);
```

Comment:
You could make `getUrlParam` type safe by using the `filter_var` method and adding a fourth argument to `getUrlParam` with the desired type, eg. `FILTER_VALIDATE_INT` or `FILTER_VALIDATE_FLOAT`.